src/daemon/ca/manager.rs

```diff
@@ -1685,7 +1685,7 @@ impl CaManager {
             .ca_objects_store
             .ca_objects(ca_handle)?
             .closest_next_update()
-            .unwrap_or_else(|| Timestamp::now_plus_hours(self.config.republish_hours()));
+            .unwrap_or_else(|| self.config.issuance_timing.republish_worst_case().into());
```

Comment:
Comparing the old code and new code here is a bit tricky, but it looks to me like the new code introduces the use of jitter, is that correct and if so is that intentional?

Reply:
yeah, this was missed earlier. As the fn name now saus this is a worst case guess for the next time, in the remote event that the current of objects is empty.. it can't really happen but I thought it would be better than unwrapping.